modules/compute/gke-node-pool/variables.tf

```diff
@@ -24,6 +24,12 @@ variable "cluster_id" {
   type        = string
 }
 
+variable "gke_cluster_exists" {
```

Comment:
is this automatically set when we have a gke-cluster or pre-existing-gke-cluster?

Reply:
Yup, gke-cluster or pre-existing-gke-cluster both have `gke_cluster_exists` as a module output.